src/zkevm_specs/evm/table.py

```diff
@@ -317,29 +317,29 @@ class CopyDataTypeTag(IntEnum):
     RlcAcc = auto()
 
 
-class MPTProofType(IntEnum):
+class ProofType(IntEnum):
     """
     Tag for MPT lookup.
     """
 
-    NonceMod = 1
-    BalanceMod = 2
-    CodeHashMod = 3
-    NonExistingAccountProof = 4
-    AccountDeleteMod = 5
-    StorageMod = 6
-    NonExistingStorageProof = 7
+    NonceChanged = 1
+    BalanceChanged = 2
+    CodeHashExists = 3
```

Comment:
Although the circuits use `CodeHashExists` I'm not sure why.  Also I would say that the canonical reference for MPT related stuff should be the MPT spec + implementation.  It's not yet merged to main/master but it uses the old names:
See specs here: https://github.com/privacy-scaling-explorations/zkevm-specs/blob/21640e4834a5eb36fc2fb4f4108520d1c4f4f655/specs/mpt/mpt-proof.md
See circuit code here: https://github.com/Brechtpd/zkevm-circuits/blob/b13b1bdd0c4cfa336ed07a915c7b64a65ff4ac9d/zkevm-circuits/src/mpt_circuit/columns.rs#L16-L23

So I think it would be better to update the zkevm-circuits State Circuit code instead of this, and keep the old names here.

As of the class name, even though the circuits type is called `ProofType`, I also think it should be `MPTProofType`, because it's defined in the table namespace, which is not specific to MPT, so I think naming it `MPTProofType` helps understand it better; otherwise seeing `ProofType` may not be clear what kind of proof type this is.